tests/generic_config_updater/test_ntp.py

```diff
@@ -107,7 +107,7 @@ def check_ntp_activestate(duthost, systemd_service):
         return False
 
     output = duthost.command(f"systemctl show {systemd_service} --timestamp unix -P ExecMainStartTimestamp")
-    return datetime.datetime.utcfromtimestamp(int(output['stdout'][1:])) > start_time
+    return int(output['stdout'][1:]) > start_time.timestamp()
```

Comment:
return int(output['stdout'][1:]) > start_time.timestamp(), the "start_time" seems to be from the local system, while output is from the duthost, why compare these two values?

Reply:
I aggree with you, maybe we should have the timestamp of the chrony restart, and then check the timpestamp of the restarted chrony. 

This need to confirm with the test case owner @wen587 and @saiarcot895

Reply:
@wen587 @saiarcot895 Could you please help to confirm, thanks a lot

Reply:
Hi @saiarcot895 , could you check this snippet? This is updated in your last PR.

Reply:
My changes here were to maintain the intention of the code that was already here, but make it cleaner to get the same information.

I don't know why this logic of checking the start time was added, if there was an issue seen that needed this logic or something else. In my opinion, if restarting ntpd/chrony via `systemctl`, then the best/cleanest way is to verify that the state of the service is active, something that `check_ntp_activestate` is already doing; the timestamp comparison shouldn't be needed at all.

Thoughts, @wen587?

Reply:
I don't think so. `active` doesn't mean it has been restart. It could be active without restart throughout the change which is not expeted.

Reply:
Is there a case where `systemctl restart ...` didn't result in at least an attempted restart (where the current running process at least exited)?

Reply:
> Is there a case where `systemctl restart ...` didn't result in at least an attempted restart (where the current running process at least exited)?

It is based on a subscribe backend service when table update. I think it is possible the backend service has some issue and didn't trigger restart when table content change..

Reply:
Ahh, my bad, I thought we were directly calling `systemctl restart ...` in this test script. In that case, yeah, process start timestamp check is the most reliable (with the possible option of checking that the config file has been updated as well).